cpp/examples/playback-ws-server/src/main.cpp

```diff
@@ -0,0 +1,257 @@
+#include <foxglove/channel.hpp>
+#include <foxglove/foxglove.hpp>
+#include <foxglove/mcap.hpp>
+#include <foxglove/server.hpp>
+
+#include <algorithm>
+#include <atomic>
+#include <chrono>
+#include <cmath>
+#include <csignal>
+#include <cstdint>
+#include <functional>
+#include <iostream>
+#include <thread>
+
+using namespace std::chrono_literals;
+
+// Example class for playing back a fixed interval of robot stack data.
+//
+// More practical implementations would load data from a file on disk, but for the sake of
+// illustration, this example generates a fixed buffer of example data in-memory.
+class DataPlayer {
+public:
+  explicit DataPlayer(size_t num_timesteps, foxglove::RawChannel&& channel)
```

Comment:
Don't think you need explicit here

Reply:
Good catch, this was left over from when the channel lived outside the class